scripts/SpecifiedDemandProfile.py

```diff
@@ -41,10 +42,13 @@ def main():
     ###########################################
 
     #generate master load dataframe
-    dfLoad = getLoadValues()
+    dfLoad = functions.getLoadValues()
 
-    #save load dataframe for use in other scripts
-    dfLoad.to_csv('../dataSources/ProvincialHourlyLoads_TimeAdjusted_AUTO_GENERATED.csv', index=False)
+    ###########################################

```

Comment:
Lets just remove this comment block. Continued from our conversation earlier, I think it was my fault on saying this auto_gen file was created in this script.